hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -178,6 +178,19 @@ <h3 style="margin-top:40px"> NetCDF Header Info</h3>
             {% endfor %}
     {% endif %}
 
+
+    {# Add netcdf opendap button  #}
+    {% if opendap_link %}
+        {% for f in cm.files.all %}
+            {% if f.resource_file.name|slice:"-3:" == '.nc'%}
+                <td><a style="margin-left:15px" id="btn-netcdf-opendap" type="button" class="btn btn-success" href='{{opendap_link }}'
+                       title='When changing this resource to or from public or private,please allow 2 minutes for the OPeNDAP service to refresh on this page' >
```

Comment:
I have checked if there is a netCDF file in the back end. So if there is no netCDF file in the resource, the button won't be shown on the resource landing page